### DISCUSSION TITLE
How to approve entries from pending list using regexp?

### DISCUSSION THREAD

**Discussion by UltraHKR** [Flexget/Flexget#3627] [Q&A]
As the title says, I want to approve entries using regexp, I'm using the config below, but the number of entries is becoming far too large to humanly manage it in a timely manner....

The idea, grab all entries from RSS, dump them in "Pending list: Queue", then task 3 can approve some entries based on a regex list that i can more easily edit to my tastes and have another task to download them...

tasks:
  RSS:
    priority: 1
    rss: https:/xyz.com/rss
    seen: local
    accept_all: yes
    list_add:
    - pending_list: Queue

  Approved:
    priority: 2
    pending_list: Queue
    regexp:
      accept:
      - sometitle
      from: Queue
  
  Download:
    priority: 3
    pending_list: Queue
    list_match:
      from:
      - pending_list: Queue
    download: /data

schedules:
  - tasks: 'RSS'
    interval:
      minutes: 60
      jitter: 60
  - tasks: 'Download'
    interval:
      minutes: 1
  - tasks: 'Approved'
    interval:
      minutes: 10

**Comment by BrutuZ** [comment]
Any reason not to do it in a single task?
```yaml
do-everything:
  rss: https://fancy.rss
  regexp:
    accept:
      - Cool Title
  download: /data
```

**Reply by UltraHKR** [reply]
Mainly because it's my first time using flexget
(I was looking for a way to archive RSS entries / files) 

I want to create different RSS lists from different RSS providers / categories 

I want to keep a history of entries (but without search in flexget it's unusable)

So what I will do later on is approve everything I have now (over 6000 entries)

And setup a flexible way to move the downloaded entries into folders... So I can manually or in a automated way filter what I need.

Basically the files (ISO torrents) should then be moved into an incoming folder on the desired client I have different clients per category.

**Comment by gazpachoking** [comment]
pending plugin is for manual intervention. If you are doing automated steps, no need to involve it. If you need the intermediary list for automated action, entry_list is available
